modular_skyrat/modules/customization/modules/client/preferences.dm

```diff
@@ -2285,6 +2287,22 @@ GLOBAL_LIST_INIT(food, list(
 						pref_scream = new scream
 						SEND_SOUND(user, pick(pref_scream.male_screamsounds))
 
+				if("laugh")
+					var/list/available_laughs = list()
+					for(var/spath in subtypesof(/datum/laugh_type)) //We need to build a custom list of available laughs!
+						var/datum/laugh_type/laugh = spath
+						if(initial(laugh.restricted_species_type))
+							if(!istype(pref_species, initial(laugh.restricted_species_type)))
+								continue
+						if(initial(laugh.donator_only) && !GLOB.donator_list[parent.ckey] && !check_rights(R_ADMIN, FALSE))
+							continue
+						available_laughs[initial(laugh.name)] = spath
+					var/new_laugh_id = input(user, "Choose your character's laugh:", "Character Scream")  as null|anything in available_laughs
```

Comment:
Huh, I thought I put a line comment here earlier
The third argument for the `input()` proc should probably be "Character Laugh" instead of scream

Reply:
Whoops, you're entirely right.

Reply:
It's all good - got all the other stuff ahaha
I'm pretty sure it's inconsequential - afaik this arg is just the title of the popout window.

Reply:
I'll just fix it in a later PR I guess. no big deal.